static-build/cmake/Package.cmake

```diff
@@ -0,0 +1,89 @@
+# CPack helps to create packages for deb and rpm based distros. 
```

Comment:
1. checkpatch fails in CI.
2. No changelog entry (what does 'NO_CHANGELOG=draft' mean if the PR is sent for review?).